var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -299,8 +299,8 @@ class PyTensorflow(Package, CudaPackage, ROCmPackage):
     depends_on("protobuf@:3.17")
     # https://github.com/protocolbuffers/protobuf/issues/10051
     # https://github.com/tensorflow/tensorflow/issues/56266
-    depends_on("py-protobuf@:3.19", type=("build", "run"))
-    depends_on("protobuf@:3.19", type=("build", "run"))
+    depends_on("py-protobuf@:3.20", type=("build", "run"))
+    depends_on("protobuf@:3.20", type=("build", "run"))
```

Comment:
According to https://github.com/tensorflow/tensorflow/issues/56266, 3.20 doesn't work?